nethsm/__init__.py

```diff
@@ -29,6 +29,8 @@
 if TYPE_CHECKING:
     from .client import ApiException
     from .client.apis.tags.default_api import DefaultApi
+    from .client.components.schema.ak_pub import AkPubDict
+    from .client.components.schema.pcr import PcrDict
```

Comment:
These are not only needed for type checking but also at runtime.  But as the auto-generated types are very hard to read, I would prefer to use custom dataclasses in `SystemInfo` instead anyway.

Reply:
I moved the values to a `TPM` data class.

Reply:
Thanks!